worker-message-prioritization-distribution/src/main/java/com/github/workerframework/workermessageprioritization/redistribution/shovel/NonRunningShovelChecker.java

```diff
@@ -38,13 +38,15 @@ public final class NonRunningShovelChecker implements Runnable
     private final LoadingCache<String,RabbitManagementApi<ShovelsApi>> nodeSpecificShovelsApiCache;
     private final Map<String, Instant> shovelNameToTimeObservedInNonRunningState;
     private final String rabbitMQVHost;
+    private final String rabbitMQUri;
     private final long nonRunningShovelTimeoutMilliseconds;
     private final long nonRunningShovelTimeoutCheckIntervalMilliseconds;
 
     public NonRunningShovelChecker(
             final RabbitManagementApi<ShovelsApi> shovelsApi,
             final LoadingCache<String,RabbitManagementApi<ShovelsApi>> nodeSpecificShovelsApiCache,
             final String rabbitMQVHost,
+            final String rabbitMQUri,
```

Comment:
```suggestion
            final String rabbitAmpqUri,
```
Would this help differentiate it from the managment uri? If you agree rename elsewhere.